### ISSUE TITLE
bug: ERROR Error: Uncaught (in promise): Error: 10 : Google Sign In - Android device

### ISSUE THREAD

**Issue by Sampath-Lokuge** [robingenz/capacitor-firebase#51]
Hi,

I have followed your demo app and done all the configurations as per your docs. When I push the app to a Native Android device using this CLI it shows the below error on the console. It opens the Sign In popup and I have selected my Gmail a/C there. After that below error. Any clue, please.

`ionic cap run android -l --host=0.0.0.0`

I have tried with Android studio too. But the same error:

```
 Ionic CLI                     : 6.16.3 (C:\Users\Sampath\AppData\Roaming\npm\node_modules\@ionic\cli)
   Ionic Framework               : @ionic/angular 6.0.11
   @angular-devkit/build-angular : 13.0.1
   @angular-devkit/schematics    : 13.0.1
   @angular/cli                  : 13.0.1
   @ionic/angular-toolkit        : 5.0.0

Capacitor:

   Capacitor CLI      : 3.4.0
   @capacitor/android : 3.4.1
   @capacitor/core    : 3.4.0
   @capacitor/ios     : not installed

Utility:

   cordova-res : 0.15.3
   native-run  : 1.5.0

System:

   NodeJS : v14.16.1 (C:\Program Files\nodejs\node.exe)
   npm    : 6.14.12
   OS     : Windows 10
```

```
main.5dfd8e110055e9fe.js:1 ERROR Error: Uncaught (in promise): Error: 10: 
Error: 10: 
    at Object.cap.fromNative (VM3:411:32)
    at <anonymous>:1:18
    at A (polyfills.399b2c612e14986c.js:1:17320)
    at polyfills.399b2c612e14986c.js:1:16402
    at Ie (main.5dfd8e110055e9fe.js:1:195570)
    at h.invoke (polyfills.399b2c612e14986c.js:1:7321)
    at Object.onInvoke (main.5dfd8e110055e9fe.js:1:336317)
    at h.invoke (polyfills.399b2c612e14986c.js:1:7261)
    at h.run (polyfills.399b2c612e14986c.js:1:2414)
    at polyfills.399b2c612e14986c.js:1:18126
    at h.invokeTask (polyfills.399b2c612e14986c.js:1:8006)
    at Object.onInvokeTask (main.5dfd8e110055e9fe.js:1:336133)
D_ @ main.5dfd8e110055e9fe.js:1
handleError @ main.5dfd8e110055e9fe.js:1
next @ main.5dfd8e110055e9fe.js:1
__tryOrUnsub @ main.5dfd8e110055e9fe.js:1
next @ main.5dfd8e110055e9fe.js:1
_next @ main.5dfd8e110055e9fe.js:1
next @ main.5dfd8e110055e9fe.js:1
next @ main.5dfd8e110055e9fe.js:1
emit @ main.5dfd8e110055e9fe.js:1
(anonymous) @ main.5dfd8e110055e9fe.js:1
h.invoke @ polyfills.399b2c612e14986c.js:1
h.run @ polyfills.399b2c612e14986c.js:1
runOutsideAngular @ main.5dfd8e110055e9fe.js:1
onHandleError @ main.5dfd8e110055e9fe.js:1
h.handleError @ polyfills.399b2c612e14986c.js:1
h.runGuarded @ polyfills.399b2c612e14986c.js:1
f @ polyfills.399b2c612e14986c.js:1
t.microtaskDrainDone @ polyfills.399b2c612e14986c.js:1
m @ polyfills.399b2c612e14986c.js:1
Promise.then (async)
g @ polyfills.399b2c612e14986c.js:1
h.scheduleTask @ polyfills.399b2c612e14986c.js:1
onScheduleTask @ polyfills.399b2c612e14986c.js:1
h.scheduleTask @ polyfills.399b2c612e14986c.js:1
h.scheduleTask @ polyfills.399b2c612e14986c.js:1
h.scheduleMicroTask @ polyfills.399b2c612e14986c.js:1
H @ polyfills.399b2c612e14986c.js:1
A @ polyfills.399b2c612e14986c.js:1
(anonymous) @ polyfills.399b2c612e14986c.js:1
cap.fromNative @ VM3:428
(anonymous) @ VM39:1
```

**Comment by robingenz** [comment]
Hi @Sampath-Lokuge,
i close this as duplicate of https://github.com/robingenz/capacitor-firebase-authentication-demo/issues/164.
Please have a look at this comment: https://github.com/robingenz/capacitor-firebase-authentication-demo/issues/164#issuecomment-1081776082

**Comment by Sampath-Lokuge** [comment]
Yes, my issue was the SHA1 key. 

When I use this `keytool -list -v -alias androiddebugkey -keystore %USERPROFILE%\.android\debug.keystore`

It always generates the same SHA1 key for any project. So I have realized that was the issue here. After that, I have used Android Studio for generating SHA1 key like so: https://stackoverflow.com/a/67866731/1077309

After that, I deleted the old one on the Firebase console and add a new one and all are working fine now. 

Thanks a lot for the top-quality plugin, docs, and demo app. Keep up the great work!
